### PR TITLE
docs update for rule yaml naming changes

### PR DESCRIPTION
**Pull request by GregHanson**

### CODE REVIEW

File: _docs/4-kubernetes/1-controlplane.md [hunk 1]
```diff
@@ -17,37 +17,52 @@ order: 2
 
 ### Deploying Amalgam8 Control Plane <a id="deploy"></a>
 
-Amalgam8 rus a control-loop to validate routing rules stored in Third Party Resources (TPR)
+Amalgam8 runs a control-loop to validate routing rules stored in Third Party Resources (TPR)
```
Comment (ijsnellf) [inline _docs/4-kubernetes/1-controlplane.md]: This is inaccurate. The controller validates incoming rules before storing them as TPRs.

File: _docs/4-kubernetes/2-helloworld.md [hunk 2]
```diff
@@ -82,55 +83,54 @@ Please refer to [helloworld](/docs/demo-helloworld.html) for a detailed descript
...
+   $ a8ctl rule-create -f examples/helloworld-default-route-rules.json
    ```
 
 1. We can confirm the rule is set by running the following command and confirming that `Status.state` is set to `valid`:
 
    ```bash
    $ kubectl get routingrule
-   NAME                        KIND
-   set-helloworld-default-v1   RoutingRule.v1.amalgam8.io
+    NAME                             KIND
```
Comment (ijsnellf) [inline _docs/4-kubernetes/2-helloworld.md]: `a8ctl rule-get -a -o json`

File: _docs/4-kubernetes/3-bookinfo.md [hunk 3]
```diff
@@ -100,22 +100,47 @@ Please refer to [bookinfo](/docs/demo-bookinfo.html) for a detailed description
...
-   $ kubectl get routingrule set-reviews-v2-user-jason -o json
-   {
-      "apiVersion": "amalgam8.io/v1",
-      "kind": "RoutingRule",
-      ...
-      "status": {
-         "state": "valid"
-      }
-   }
+   $ kubectl get routingrule bookinfo-jason-reviews-v2-route-rules -o json
```
Comment (ijsnellf) [inline _docs/4-kubernetes/3-bookinfo.md]: `a8ctl rule-get -i bookinfo-jason-reviews-v2-route-rules -o json`

**Review by ijsnellf — CHANGES_REQUESTED** [review]

File: _docs/4-kubernetes/2-helloworld.md (hunk 2 above)
Comment (GregHanson) [reply _docs/4-kubernetes/2-helloworld.md]: @rshriram @elevran do we just want to get rid of the entire `4-kubernetes` section?  Assuming the docs in `1-demo` cover everything kubernetes related?

Comment (rshriram) [reply _docs/4-kubernetes/2-helloworld.md]: Yes..

Comment (rshriram) [reply _docs/4-kubernetes/2-helloworld.md]: Well, make sure that there all info in the k8s section is covered in demo. Secondly, I would take the description on TPR stuff and stick it into the controller page as an implementation note

**Review by ijsnellf — APPROVED** [review]

File: _docs/2-controlplane/1-controller/configuration-options.md [hunk 4]
```diff
@@ -61,3 +62,52 @@ with a password, set the options above and this additional option as well:
...
+      labels:
+        name: controller
+    spec:
+      containers:
+      - name: controller
+        image: amalgam8/a8-controller
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: A8_DATABASE_TYPE
+          value: kubernetes
```
Comment (rshriram) [inline _docs/2-controlplane/1-controller/configuration-options.md:104]: k8s or kubernetes?

**Review by rshriram — APPROVED** [review]

File: _docs/2-controlplane/1-controller/configuration-options.md (hunk 4 above)
Comment (GregHanson) [reply _docs/2-controlplane/1-controller/configuration-options.md:104]: according etai, our naming convention has been `kubernetes` for similar environment variables

**Pull request merged**